File: benchmarks/crosslang/node/fib30.js
const { performance } = require("perf_hooks");
function fib(n) {
  if (n <= 1) return n;
  return fib(n - 1) + fib(n - 2);
}
const s = performance.now();
const result = fib(30);
const e = performance.now();
console.log(result);
console.log(e - s);

File: benchmarks/crosslang/lua/fib30.lua
local function fib(n)
  if n <= 1 then return n end
  return fib(n - 1) + fib(n - 2)
end
local s = os.clock()
local result = fib(30)
local e = os.clock()
print(result)
print((e - s) * 1000.0)
